DEVELOPMENT.md

```diff
@@ -195,3 +195,43 @@ To access Telemetry see:
 - [Accessing Metrics](https://www.knative.dev/docs/serving/accessing-metrics/)
 - [Accessing Logs](https://www.knative.dev/docs/serving/accessing-logs/)
 - [Accessing Traces](https://www.knative.dev/docs/serving/accessing-traces/)
+
+## Packet sniffing
+
+While debugging an Eventing component, it could be useful to perform packet sniffing on a container to analyze the traffic.
```

Comment:
<!--mdfmt-bot-->
Format markdown:
```suggestion
While debugging an Eventing component, it could be useful to perform packet
sniffing on a container to analyze the traffic.
```